optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/drools/DroolsConstraint.java

```diff
@@ -77,6 +72,6 @@ public int getExpectedJustificationCount() {
 
     @Override
     public String toString() {
-        return "DroolsConstraint(" + getConstraintId() + ") in " + fromStreamList.size() + " from() stream(s)";
+        return "DroolsConstraint(" + getConstraintId() + ")";
```

Comment:
The only use of `fromStreamList()` was in this `toString()`.
Therefore I removed it.